css/custom.css

```diff
@@ -1,4 +1,5 @@
 @import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
```

Comment:
Maybe not ideal for an extension to load new fonts. I don't know if we are using the firts one, but I would like to discuss this. I believe that packing the font file directly into the extension bundle would be benefice.